content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: पुल अनुरोधों की समीक्षा करना
+content_type: concept
+main_menu: true
+weight: 10
+---
+
+<!-- अवलोकन -->
+
+कोई भी प्रलेखन पुल अनुरोध की समीक्षा कर सकता है। खुले पुल अनुरोधों को देखने के लिए
+कुबेरनेट्स वेबसाइट रिपॉजिटरी में [पुल अनुरोध](https://github.com/kubernetes/website/pulls) अनुभाग पर जाएँ।
+
+
+प्रलेखन पुल अनुरोधों की समीक्षा करना कुबेरनेट्स समुदाय से अपना परिचय कराने का एक शानदार तरीका
+है. यह आपको कोड बेस सीखने और अन्य योगदानकर्ताओं के साथ विश्वास बनाने में मदद करता है।
+
+समीक्षा करने से पहले, यह एक अच्छा विचार है:
```

Comment:
```suggestion
रिव्यु करने से पहले, यह एक अच्छा विचार है:
```

Reply:
I would stick to समीक्षा and would not transliterate review.